modules/mediawiki/files/bin/deploy-mediawiki.py

```diff
@@ -33,12 +33,34 @@ def run(args, start):
         os.system(f'/usr/local/bin/logsalmsg {text}')
     else:
```

Comment:
```suggestion
    else:
        print(text)
    if args.pull:
        pull = str(args.pull).split(',')
        if args.world not in pull:
            pull.append('world')
        if 'config' in pull:
```

Reply:
This wouldn't work either, as then we are always pulling even if args.world isn't used. Though I guess `if args.world and 'world' not in pull:` might work?